src/pylero/document.py

```diff
@@ -572,3 +572,37 @@ def update(self):
             Tracker.updateModule
         """
         self.session.tracker_client.service.updateModule(self._suds_object)
+
```

Comment:
should the revision objects be translated to Pylero objects and be storage in suds_map?

Reply:
I am thinking of moving those methods up to the BasePolarion base class since we need this functionality also for WikiPage , WDYT ?

Reply:
Ack, that also works, I believe these revisions doesn't have any set methods and so entries in suds_map isn't required.